org.adempiere.base/src/org/compiere/process/ProcessInfoParameter.java

```diff
@@ -282,4 +285,78 @@ public void setParameterName (String ParameterName)
 		m_ParameterName = ParameterName;
 	}
 
+	/**
+	 * Return the value of the parameter as a validated String (all values between commas must be integer)
+	 * @return String

```

Comment:
I think this can be changed to "Return the value of the parameter as a comma separated integer string. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.".

Also, I think there might be instance where you would want an int[] instead of a csv string.

Reply:
done